packages/core/src/toggle-button/ToggleButton.css

```diff
@@ -19,7 +19,7 @@
   font-family: var(--salt-text-action-fontFamily);
   letter-spacing: var(--salt-text-action-letterSpacing);
   text-transform: var(--salt-text-action-textTransform);
-  padding: 0 calc(var(--salt-spacing-100) - var(--salt-size-border, 0));
+  padding: 0 calc(var(--salt-spacing-100) - var(--salt-size-fixed-100, 0));
```

Comment:
as this is padding should it be spacing-fixed instead of size-fixed?

Reply:
This is an "inside border" calculation so it's referring to the width of the border